### PR TITLE
rtl8730e_boot.c: bringup gpio PB22 pin for lcd

### PR DESCRIPTION
**Pull request by ritesh55555**
This commit brings up PB22 gpio pin. This pin will be used as wake up source for lcd.

### CODE REVIEW

File: os/board/rtl8730e/src/rtl8730e_boot.c [hunk 1]
```diff
@@ -242,6 +242,7 @@ void board_gpio_initialize(void)
 				{PA_23, PIN_INPUT, PullNone},
 				/* PB_20 is gpio pin number for LED */
 				{PB_20, PIN_OUTPUT, PullDown},
+				{PB_22, PIN_INPUT, PullNone},
```
[the remark this answers is not in the document]
Comment (sunghan-chang) [reply os/board/rtl8730e/src/rtl8730e_boot.c:245]: My comment is wrong. Sorry.

File: os/board/rtl8730e/src/rtl8730e_boot.c [hunk 2]
```diff
@@ -242,6 +242,7 @@ void board_gpio_initialize(void)
 				{PA_23, PIN_INPUT, PullNone},
 				/* PB_20 is gpio pin number for LED */
 				{PB_20, PIN_OUTPUT, PullDown},
+				{PB_22, PIN_OUTPUT, PullNone},
```
Comment (sunghan-chang) [inline os/board/rtl8730e/src/rtl8730e_boot.c:245]: 1. It should be PIN_INPUT. (My previous comment was wrong)
2. Let me suggest the title like below
```
board/rtl8730e: set PB_22 GPIO pin to get outside wakeup event

The PB_22 pin will be registered as a wakeup source and will get a wakeup event from outside.
```

Comment (ewoodev) [inline os/board/rtl8730e/src/rtl8730e_boot.c:245]: os/arch/arm/src/amebasmart/amebasmart_i2s.c
	.i2s_mclk_pin = PB_22,

PB22 pin is setting I2S2 mclk. 
We need to check first if we can change I2S2 mclk to un-used pin

Comment (ewoodev) [reply os/board/rtl8730e/src/rtl8730e_boot.c:245]: rtk will raise PR about I2S2,
@ritesh55555 Let's merge your PR, Please update

Comment (ritesh55555) [reply os/board/rtl8730e/src/rtl8730e_boot.c]: done

**Review by ewoodev — APPROVED** [review]

**Review by r-prabu — APPROVED** [review]

**Pull request merged**